clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -7377,6 +7377,18 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
 
   Args.AddLastArg(CmdArgs, options::OPT_finline_max_stacksize_EQ);
 
+  // Adaptation of partial-inlining option with GCC.
+  if (Arg *A = Args.getLastArg(options::OPT_fno_partial_inlining,
```

Comment:
Also... this seems to be missing the 'reverse' thing that we have to do?  Perhaps I've not been in the driver for a while, but we have a thing where during asserts mode we have to un-do the flags and re-form them, and then it checks to see if they are the same.